functests/README.txt

```diff
@@ -0,0 +1,5 @@
+HEADS UP!
+
+We have 2 test suites here, one running the "normal" performance tests, the other one performance profile update tests.
+The latter should run AFTER the former, and tests are executed in order of filenames.
```

Comment:
Which would mean if I create test_suite_performance_status_test.go it would get executed before test_suite_performance_update_test.go right? The status tests requires the MCP to be in degraded state, hence it would be better to execute these tests at the end.

Reply:
Yes. Maybe I should add a number into the name like 
- test_suite_1_performance_test.go
- test_suite_2_performance_update_test.go
- test_suite_3_performance_status_test.go

to make it more explicit?

Reply:
I had to move the test suites into the subdirectories, so I added numbers to the directory name